versioned_docs/version-1.2.x/surrealql/statements/define/field.mdx

```diff
@@ -80,21 +83,34 @@ DEFINE FIELD posts.* ON TABLE user TYPE record<string>;
 ```
 ### Making a field optional
 
+You can make a field optional by using the `option` type. This allows you to store `NONE` values in the field.
```

Comment:
This might be a bit Rust-like terminology but I've always found "by wrapping the inner type in an `option`" to be a nice way to describe it. (Also looks like option doesn't work without the inner type so I think it could describe the difference between it and the default `any` type with other fields)

Reply:
Ah! Yes it does read better with this suggestion. Updated now.